spec/app/repos_spec.rb

```diff
@@ -0,0 +1,49 @@
+RSpec.describe Repos do
+  before :each do
+    allow(Repos).to receive(:all) do
+      applications.map(&:stringify_keys).map { |app_data| App.new(app_data) }
+    end
+  end
+
+  describe "public" do
```

Comment:
It looks like this is halfway to using the conventional define method approach for RSpec: https://www.betterspecs.org/#describe so this should be `describe ".public"` Looking at the source code I think this project is almost using the regular `describe "#instance_method"` or `describe ".class_method"` style so we probably want to bring this file into line.